packages/component/src/Composer.js

```diff
@@ -113,31 +113,26 @@ function createFocusSendBoxContext({ sendBoxRef }) {
 }
 
 // TODO: [P3] Take this deprecation code out when releasing on or after 2019 December 11
-function patchPropsForAvatarInitials({ botAvatarInitials, userAvatarInitials, ...props }) {
-  // This code will take out "botAvatarInitials" and "userAvatarInitials" from props
-
-  let { styleOptions } = props;
+function patchStyleOptionsForAvatarInitials({ botAvatarInitials, styleOptions, userAvatarInitials }) {
+  let patchedStyleOptions = { ...styleOptions };
 
   if (botAvatarInitials) {
```

Comment:
Should we just remove `botAvatarInitials` and `userAvatarInitials` since I think the next release is after 12/11?

Reply:
Done.